src/web/services/feedback-sessions.service.ts

```diff
@@ -399,8 +400,10 @@ export class FeedbackSessionsService {
    */
   generateNameFragment(feedbackSession: FeedbackSession): string {
     const DATE_FORMAT_WITH_ZONE_INFO: string = 'ddd, DD MMM yyyy, hh:mm A Z';
-    const startTime: string = moment(feedbackSession.submissionStartTimestamp).format(DATE_FORMAT_WITH_ZONE_INFO);
-    const endTime: string = moment(feedbackSession.submissionEndTimestamp).format(DATE_FORMAT_WITH_ZONE_INFO);
+    const startTime: string = this.timezoneService
+        .formatToString(feedbackSession.submissionStartTimestamp, null, DATE_FORMAT_WITH_ZONE_INFO);
```

Comment:
I believe the timezone here is in `feedbackSession.timeZone` or there will be a display bug. Let's fix it.